samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/smp_svr.c

```diff
@@ -0,0 +1,73 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_MCUMGR)
```

Comment:
This is not how zephyr manages whether sources get compiled. This must be done in the build system by using zephyr_sources_ifdef(), zephyr_sources_if_kconfig(), or similar.

Reply:
Yes, I tried this. But as I already mentioned in last some comments that after it we can't build the app. Some include files are not get properly linked & this is cause behind it.

Reply:
Then you need to figure out why 😉

Reply:
Finally, I have figured out !!!!